### PR TITLE
chore: contributing updates

### PR DESCRIPTION
**Pull request by brenkao**
* Updated github issues to include emoji :) and make it more clear questions should be asked in GitHub discussions.
* Updated contributing to be more clear on how to test.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml
Comment (willbakst) [inline .github/ISSUE_TEMPLATE/config.yml:1]: hmm, I would rather people post in discussions than post an issue with a question. Thoughts?

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
...
     ```
 
-4.  Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+4. Test!
 
-    - When you are ready to submit your contribution for review, push your branch:
+    - Run tests to make sure nothing broke
+
+    ```shell
+    poetry run pytest
```
Comment (willbakst) [inline docs/CONTRIBUTING.md:74]: update this to `poetry run pytest tests/`

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
     poetry run mypy .
     ```
 
-4.  Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+4. Test!
```
Comment (willbakst) [inline docs/CONTRIBUTING.md:69]: I think first we should mention that they need to write tests and where to write them? i.e. add tests to the tests/ directory, which matches the structure of mirascope/ etc.

There is already a Testing section below, maybe we should pull that up here and merge it into this section?

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
...
+
+    - Check coverage report
+
+    ```shell
+    poetry run pytest tests/ --cov=./ --cov-report=html
+    ```
+
+5. Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+    - When you are  ready to submit your contribution for review, push your branch:
```
Comment (willbakst) [inline docs/CONTRIBUTING.md:85]: somewhere in this section add that they should make sure to check the box to allow edits from maintainers

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
...
 
     ```shell
     git push origin meaningful-branch-name
```
Comment (willbakst) [inline docs/CONTRIBUTING.md:88]: it might also be worth noting below that you can submit the PR by using the Contribute button/feature in the GitHub web page for the repo fork

File: .github/ISSUE_TEMPLATE/config.yml
Comment (brenkao) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Thats what this is

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (brenkao) [reply docs/CONTRIBUTING.md:69]: that section is too wordy to be bullet point, i will add where to write tests and colocation

File: .github/ISSUE_TEMPLATE/config.yml
Comment (willbakst) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Ah I see that now

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (brenkao) [reply docs/CONTRIBUTING.md:85]: done

File: docs/CONTRIBUTING.md (hunk 4 above)
Comment (brenkao) [reply docs/CONTRIBUTING.md:98]: done

**Review by willbakst — APPROVED** [review]
